vivarium/core/store.py

```diff
@@ -651,9 +657,6 @@ def _apply_config(self, config, source=None):
                     self.units = self.units or self.default[0].units
                     self.serializer = (self.serializer or
                                        serializer_registry.access('units'))
-                elif isinstance(self.default, np.ndarray):
```

Comment:
NumPy arrays are handled by the PyMongo codec system so we can avoid the slow Python code for store-specific serialization.